<ui version="4.0" >
 <class>GeneralConfig</class>
 <widget class="QWidget" name="GeneralConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>359</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Backend</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Executable:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>engineExecutable</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="KUrlRequester" name="engineExecutable" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Select the executable file to start the Go engine</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Parameters:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>engineParameters</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KLineEdit" name="engineParameters" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Add here the necessary parameters to start the Engine in GTP mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="KLineEdit" name="kcfg_EngineCommand" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>This is the resulting engine command which will be used by Kigo</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="contextMenuEnabled" >
         <bool>false</bool>
        </property>
        <property name="urlDropsEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="KLed" name="engineLed" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Indicates whether the Go engine works correctly</string>
        </property>
        <property name="state" >
         <enum>KLed::Off</enum>
        </property>
        <property name="shape" >
         <enum>KLed::Circular</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="s" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_ShowBoardLabels" >
        <property name="text" >
         <string>Display board labels</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Move hint visibility:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>kcfg_HintVisibleTime</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="kcfg_HintVisibleTime" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix" >
         <string> Seconds</string>
        </property>
        <property name="decimals" >
         <number>1</number>
        </property>
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.500000000000000</double>
        </property>
        <property name="value" >
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_AdvancedStatistics" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Show detailed statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_AdvancedGameOptions" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable expert game options</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_ShowBoardLabels</tabstop>
  <tabstop>kcfg_HintVisibleTime</tabstop>
  <tabstop>kcfg_AdvancedStatistics</tabstop>
  <tabstop>kcfg_AdvancedGameOptions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
